databuilder/tables/beta/tpp.py

```diff
@@ -115,6 +131,28 @@ class addresses(EventFrame):
     care_home_requires_nursing = Series(bool)
     care_home_does_not_require_nursing = Series(bool)
 
+    def active_for_patient_at(self, date):
```

Comment:
we can name this function the same as in line 62 because databuilder knows whether the function gets called on address or registration data?

Reply:
I see, yes that makes sense and it'd be nice to keep the same name for these two methods